[2024-06-11T22:54:56.226+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task2 manual__2024-06-11T22:54:54.993559+00:00 [queued]>
[2024-06-11T22:54:56.231+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task2 manual__2024-06-11T22:54:54.993559+00:00 [queued]>
[2024-06-11T22:54:56.231+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:54:56.231+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-11T22:54:56.231+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:54:56.240+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task2> on 2024-06-11 22:54:54.993559+00:00
[2024-06-11T22:54:56.243+0000] {standard_task_runner.py:55} INFO - Started process 4533 to run task
[2024-06-11T22:54:56.245+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task2', 'manual__2024-06-11T22:54:54.993559+00:00', '--job-id', '243', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmp5doqgjiu']
[2024-06-11T22:54:56.246+0000] {standard_task_runner.py:83} INFO - Job 243: Subtask task2
[2024-06-11T22:54:56.276+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task2 manual__2024-06-11T22:54:54.993559+00:00 [running]> on host 91b1d49ba724
[2024-06-11T22:54:56.309+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task2
AIRFLOW_CTX_EXECUTION_DATE=2024-06-11T22:54:54.993559+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-11T22:54:54.993559+00:00
[2024-06-11T22:54:56.309+0000] {logging_mixin.py:120} INFO - Task 2 is completed
[2024-06-11T22:54:57.312+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-11T22:54:57.337+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task2, execution_date=20240611T225454, start_date=20240611T225456, end_date=20240611T225457
[2024-06-11T22:54:57.362+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-11T22:54:57.377+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
